apps/src/gamelab/dropletConfig.js

```diff
@@ -13,11 +14,23 @@ var COLOR_WHITE = '#FFFFFF';
 var COLOR_BLUE = '#64B5F6';
 var COLOR_ORANGE = '#FFB74D';
 
+var GameLab;
+
+exports.injectGameLab = function (gamelab) {
+  GameLab = gamelab;
+};
+
+// Flip the argument order so we can bind `typeFilter`.
+function chooseAsset(typeFilter, callback) {
+  showAssetManager(callback, typeFilter);
```

Comment:
This is fine, I guess.  You could always use [lodash `partial` with placeholders](https://lodash.com/docs#partial), or [lodash `flip`](https://lodash.com/docs#flip).